src/GitExtensions.Extensibility.Tools/tools/Download-GitExtensions.ps1

```diff
@@ -0,0 +1,122 @@
+param([string] $ExtractRootPath, [string] $Version, [string] $Source = "github")
+
+if (-not($ExtractRootPath)) 
+{ 
+    Throw "Parameter -ExtractRootPath is required";
+}
+
+if (-not($Version)) 
+{ 
+    Throw "Parameter -Version is required";
+}
+
+if (-not($Source -eq "github") -and -not($Source -eq "appveyor"))
+{
+	Throw "Parameter -Source must be either 'github' or 'appveyor'";
+}
+
+Set-Location $PSScriptRoot
+
+$ExtractRootPath = Resolve-Path $ExtractRootPath;
+Write-Host "Extraction root path is '$ExtractRootPath'.";
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$AssetToDownloadName = 'GitExtensions-' + $Version + '.zip';
+$AssetToDownloadUrl = $null;
+
+$DownloadName = [System.IO.Path]::GetFileName($AssetToDownloadName);

```

Comment:
Just FYI, you could also use [`Split-Path`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/split-path?view=powershell-6)

Reply:
Completely removed.